src/cmds/scripts/pbs_postinstall.in

```diff
@@ -375,25 +375,6 @@ install_pbsinitd() {
 				fi
 			fi
 			;;
-		Darwin)
-			dir="/Library/StartupItems/PBS"
-			initscript="$dir/PBS"
-			createdir "$dir" 0755
-			cp ${PBS_EXEC}/libexec/pbs_init.d $initscript
-			cat <<-EOF > $dir/StartupParameters.plist
-				{
-				  Description     = "Portable Batch System";
-				  Provides        = ("PBS");
-				  Requires        = ("Network");
-				  OrderPreference = "None";
-				  Messages =
-				  {
-				    start = "Starting PBSPro";
-				    stop  = "Stopping PBSPro";
-				  };
-				}
-			EOF
-			;;
```

Comment:
Why is the Darwin case being removed. This was working some years back, the last time I modified it.

Reply:
Are we supporting Darwin in PBS? @scc138 ?

Reply:
The short answer is no. It was needed for an internal project a few years back.

Reply:
@mike0042 so now don't need it now right?

Reply:
As far as I know, it's no longer needed. Just seemed that the removal was out of scope for this PR.

Reply:
As you can see there were PBSPro word in it which I had to change to PBS, but then I thought anyways this is not supported code so I removed code instead of changing words